data/applications.yml

```diff
@@ -168,8 +168,16 @@
   api_docs_url: https://mapit.mysociety.org/docs/
 
 - github_repo_name: metadata-api
+  retired: true
   type: APIs
   team: "#survey-and-support"
+  description: |
+    API written in Go that was used to get metadata about given URLs on GOV.UK.
+    The api would communicate with the `backdrop read API` on behalf of `info-frontend`
+    and retrieve information about user needs. The `backdrop read API` would talk
```

Comment:
I am fairly sure that metadata API spoke to need API directly, not via backdrop.

Reply:
Corrected